src/block_cache/block_io.c

```diff
@@ -399,6 +420,16 @@ __wt_blkcache_write(WT_SESSION_IMPL *session, WT_ITEM *buf, uint8_t *addr, size_
     WT_STAT_SESSION_INCRV(session, bytes_write, dsk->mem_size);
     (void)__wt_atomic_add64(&S2C(session)->cache->bytes_written, dsk->mem_size);
 
+    if (dsk != NULL) {
+        if (dsk->type == WT_PAGE_COL_INT || dsk->type == WT_PAGE_ROW_INT) {
+            WT_STAT_CONN_INCRV(session, block_byte_write_intl, buf->size);
```

Comment:
nitpick: may be change buf->size to dsk->mem_size to be consistent to the previous usages.